package.json

```
@@ -2,7 +2,8 @@
   "name": "@testing-library/svelte",
   "version": "0.0.0-semantically-released",
   "description": "Simple and complete Svelte testing utilities that encourage good testing practices.",
-  "main": "dist/index.js",
+  "main": "src/index.js",
```

probably best to use `exports` as it's the newer way to specify
```suggestion
  "exports": "src/index.js",
```